pkg/config/config.go

```diff
@@ -121,20 +122,31 @@ func UseContext(context string) error {
 	return fmt.Errorf("Context %s not found", context)
 }
 
-func CurrentContext() (*Context, error) {
+func currentViperContext() *viper.Viper {
 	name := viper.GetString(CURRENT_CONTEXT)
 	if name == "" {
 		NewConfig()
-		return CurrentContext()
+		return currentViperContext()
 	}
 	contextPath := fmt.Sprintf("contexts.%s", name)
 	ctx := viper.Sub(contextPath)
 	if ctx == nil {
 		ctx = viper.New()
 	}
+
+	// Set defaults
+	if !ctx.IsSet("synthetic-monitoring.api-url") {
+		ctx.Set("synthetic-monitoring.api-url", "https://synthetic-monitoring-api.grafana.net")
```

Comment:
Hardwired value in config? If hardwired, surely that would be in the handler?

Reply:
I don't know. It's a default value to a configuration, not really hardwired. It's more visible this way, and that's why it's more easily testable.

Either way works, I can switch to figuring out the URL in code 🤔

Reply:
`viper` has `viper.SetDefault("ContentDir", "content")` that would be more appropriate here, and could happen earlier? I guess upon creation of a new context? I wonder if we could make defaults a per-handler thing, where we defer to each handler to configure its defaults. Then, for Grafana we could default to http://localhost:3000/admin/admin for example?